lib/worker.js

```diff
@@ -167,7 +168,21 @@ var HTTPListner = function() {
 				response.end();
 			}
 		},
+		'/analyze-url' : function( request, response ) {
+			const params = new URLSearchParams( request.url.split('?')[1] );
 
+			const command = {
+				url: params.get( 'url' ),
+				key: params.get( 'cache_key' ),
```

Comment:
It feels like the `cache_key` is something we should be generating our self (could it be the same as the job key?). What if the user entered a cache key for someone else's job for example? Or is this only going to be accessible to the php client?

Reply:
The endpoint in node is only accessible by the PHP client so the user can never set the cache key. Both in PHP and Node we need an identical cache key and it is generated in PHP. When there is an entry for the cache key, we never have to communicate with the node backend.

Reply:
ahh ok I understand 👍